js/util/extractProps.ts

```diff
@@ -0,0 +1,47 @@
+const pick = require('lodash.pick');
+const flatten = require('flat');
+
+export interface Component {
+  //TODO: use the real RN ones
```

Comment:
TODOs should be of the form `// :TODO: (<username of person with the most context (not necessarily author)>): <todo comment>`

Reply:
Just removed the TODO, but will keep that in mind.